stream-chat-android-client/src/main/java/io/getstream/chat/android/client/controllers/ChannelController.kt

```diff
@@ -63,7 +63,7 @@ public interface ChannelController {
     public fun banUser(targetId: String, reason: String?, timeout: Int?): Call<Unit>
 
     @CheckResult
-    public fun unBanUser(targetId: String, reason: String?, timeout: Int?): Call<Unit>
+    public fun unBanUser(targetId: String): Call<Unit>
```

Comment:
Nice catch! 
This might be a breaking change for our clients, so it's worth mentioning that in CHANGELOG. 
Maybe we should consider introducting a separate method inside `ChannelClient` and deprecate that one?

Reply:
Reflected the change in the changelog. Not sure that we need to keep the old method.